src/web/releases.rs

```diff
@@ -439,7 +389,8 @@ pub fn author_handler(req: &mut Request) -> IronResult<Response> {
                 cexpect!(req, author.nth(1)),
             )
         } else {
-            get_releases_by_author(&mut conn, page_number, RELEASES_IN_RELEASES, author)
+            // TODO: Accurate error here, the author name is invalid
+            return Err(Nope::CrateNotFound.into());
```

Comment:
I would prefer to fix all the error message issues in that method in a separate PR.

Reply:
This seems like a really bad error message for a page that used to work ... could you add a new error variant to `Nope` that has a more descriptive message?

Reply:
There are a lot of TODOs about messages here, and I'd prefer to submit another pr to fix them all.

Reply:
Ok, do you mind submitting that PR first then?

Reply:
OK. I will add the Nope.